code/game/atoms.dm

```diff
@@ -512,10 +512,11 @@ var/list/blood_splatter_icons = list()
 		overlays += blood_overlay
 
 /atom/proc/clean_blood()
-	germ_level = 0
-	if(islist(blood_DNA))
-		blood_DNA = null
-		return 1
+	if(!istype(src, /obj/effect/decal/cleanable/blood))

```

Comment:
Heresy :eyes: (but I'll fix this myself because I don't want this to wait for Kasparov to fix again)